tests/cmd/stability/pods.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/golang/glog"
+	"github.com/pingcap/tidb-operator/tests/pkg/client"
+	"k8s.io/api/admission/v1beta1"
+)
+
+type dbInfo struct {
+	IsOwner bool `json:"is_owner"`
+}
+
+// only allow pods to be delete when it is not ddlowner of tidb, not leader of pd and not
+// master of tikv.
+func admitPods(ar v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
+	glog.Infof("admitting pods")
+	httpClient := &http.Client{}
+	podResource := metav1.GroupVersionResource{Group: "", Version: "v1", Resource: "pods"}
+	if ar.Request.Resource != podResource {
```

Comment:
You have register the webhook with [pods deletion](https://github.com/pingcap/tidb-operator/pull/389/files#diff-67dacabf080ca4dc8e95d2acf12ee36bR1905). I think there is no need to verify it again,